src/Context/LoginContext.php

```diff
@@ -80,6 +81,45 @@ public function stepIAmNotLoggedIn()
      * @param string $password
      */
     public function stepILogInWith($email, $password)
+    {
+        $this->loginWith($email, $password);
+
+        // Check if MFA module is installed
+        if (!class_exists(RegisteredMethod::class)) {
+            return;
+        }
+
+        // Skip MFA registration if MFA module installed
+        $this->getMainContext()->getSession()->wait(300);
+        $page = $this->getMainContext()->getSession()->getPage();
+        $mfa = $page->find('css', '#mfa-app');
+        if (!$mfa) {
+            return;
+        }
+        $btns = $page->findAll('css', '.mfa-action-list__item .btn');
+        foreach ($btns as $btn) {
+            if ($btn->getText() !== 'Setup later') {
+                continue;
+            }
+            $btn->click();
+        }
+    }
+
+    /**
+     * @When /^I log in with "([^"]*)" and "([^"]*)" without skipping MFA$/
+     * @param string $email
+     * @param string $password
+     */
+    public function stepILogInWithWithoutSkippingMfa($email, $password)
```

Comment:
What's the scenario where we want to do that? Is this just to cover all our basis?

Reply:
There currently isn't a scenario.  We may make a future scenario where we want to behat test the MFA flow.